.dumi/theme/builtins/RefinedAntdChangelog/index.tsx

```
@@ -0,0 +1,134 @@
+      <Popover placement="rightTop" title={locale.deprecatedTitle} content={reasonContent}>
+        <BugOutlined
+          style={{
+            fontSize: token.fontSizeHeading4,
+            color: token.colorErrorText,
+          }}
+        />
+      </Popover>
+    </Flex>
+  );
+}
+function Date(props: React.PropsWithChildren) {
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)